puppet_forge.gemspec

```
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "faraday", "~> 2.0"
   spec.add_runtime_dependency "faraday-follow_redirects", "~> 0.3.0"
+  spec.add_runtime_dependency 'base64', '~> 0.2.0'
```

Nit: rubocop prefers these to be sorted alphabetically (and so do I)

---

Doesn't look sorted, probably because this uses single quotes where the rest uses double
```suggestion
  spec.add_runtime_dependency "base64", "~> 0.2.0"
```